### PR TITLE
Fix snapshot for changes to test output from Xcode 7.3

### PR DESCRIPTION
**Pull request by mfurtak**
Addresses the issue reported in #3875

The **TestSummaries.plist** output from Xcode changed in 7.3, requiring a small change to how we determine the file path for copying captured screenshots.

### CODE REVIEW

File: snapshot/lib/snapshot/collector.rb [hunk 1]
```diff
@@ -71,7 +74,11 @@ def self.attachments(containing)
     def self.check_activity(activity, to_store)
       # We now check if it's the rotation gesture, because that's the only thing we care about
       if activity["Title"] == "Set device orientation to Unknown"
-        to_store << activity["Attachments"].last["FileName"]
+        if activity["Attachments"]
+          to_store << activity["Attachments"].last["FileName"]
+        else # Xcode 7.3 has stopped including 'Attachments', so we synthesize the filename manually
+          to_store << "Screenshot_#{activity['UUID']}.png"
```
Comment (KrauseFx) [inline snapshot/lib/snapshot/collector.rb]: :+1:

**Comment by KrauseFx** [comment]
Looks great, thanks for the super fast fix :+1:

File: snapshot/lib/snapshot/collector.rb [hunk 2]
```diff
@@ -43,12 +43,15 @@ def self.fetch_screenshots(output, dir_name, device_type, launch_arguments_index
 
     def self.attachments(containing)
       UI.message "Collecting screenshots..."
-
       plist_path = Dir[File.join(containing, "*.plist")].last # we clean the folder before each run
-      UI.verbose "Loading up '#{plist_path}'..."
```
Comment (i2amsam) [inline snapshot/lib/snapshot/collector.rb]: Did you mean to drop this?

Comment (mfurtak) [inline snapshot/lib/snapshot/collector.rb]: Nope, good catch!

**Comment by asfalcone** [comment]
LGTM2!

**Pull request merged**